lib/WasmReader/BaseWasmReader.h

```diff
@@ -43,6 +43,11 @@ namespace Wasm
         WasmNode    m_currentNode;
         ModuleInfo * m_moduleInfo;
 
+        static SectionFlag sectionFlags[bSectLimit];
```

Comment:
Since section stuff is getting involved, would it simplify things to keep these together in a struct? It would also avoid the possibility of these arrays getting desynchronized.